### ISSUE TITLE
Interactivity should have a WaitForReaction method that allows waiting on specific message.

### ISSUE THREAD

**Issue by Emzi0767** [NaamloosDT/DSharpPlus#83]
Like in the title. Would allow checking who reacted, for example for contest purposes.

**Comment by Emzi0767** [comment]
Maybe make them return the reaction info while at it.

**Comment by NaamloosDT** [comment]
Did you mean these?
https://github.com/NaamloosDT/DSharpPlus/blob/master/DSharpPlus.Interactivity/InteractivityModule.cs#L161-L212

**Comment by NaamloosDT** [comment]
Though checking for a specific user would be nice to have indeed
